shared/quantum/codeql/quantum/experimental/Model.qll

```diff
@@ -1914,12 +1920,19 @@ module CryptographyBase<LocationSig Location, InputSig<Location> Input> {
       node instanceof KeyCreationCandidateAlgorithmNode
     }
 
+    NodeBase getARawValueSource() {
+      result = keyGenInstance.getRawKeyValueConsumer().getConsumer().getAGenericSourceNode()
+      or
+      result = keyGenInstance.getRawKeyValueConsumer().getConsumer().getAKnownSourceNode()
+    }
+
     override NodeBase getChild(string key) {
       result = super.getChild(key)
       or
       // [ALWAYS_KNOWN]
       key = "Output" and
       result = this.getOutputKeyArtifact()
+      //TODO: how do I output the raw key if known? If not known, it may not require/have a raw value consumer, don't output
```

Comment:
See comment above with the suggestion to add a `hasRawKeyValueConsumer` predicate.